core/domain/exp_domain.py

```diff
@@ -2014,7 +2639,16 @@ def add_states(self, state_names):
             self.states[state_name] = State.create_default_state(state_name)
 
     def rename_state(self, old_state_name, new_state_name):
-        """Renames the given state."""
+        """Renames the given state.
+
+        Args:
+            old_state_names: str. The old state name to set.
```

Comment:
I don't know what you mean by "set" here. Aren't we renaming the state from old_state_name to new_state_name?